metrics/service-name.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function () {
+  const selectors = {
+    serviceNavigationLink:
+      '.govuk-service-navigation__service-name .govuk-service-navigation__link',
+    serviceNavigationText:
+      '.govuk-service-navigation__service-name .govuk-service-navigation__text',
+    headerServiceName: '.govuk-header__service-name',
+    headerServiceNameLink: '.govuk-header__link--service-name',
+    headerProductName: '.govuk-header__product-name',
+    title: 'head title'
+  }
+
+  for (const selector of Object.values(selectors)) {
+    const element = document.querySelector(selector)
+    if (element) {
+      return element.textContent.trim()
+    }
+  }
+
+  return 'unknown'
```

Comment:
This is something we'd learn more about during implementation but what would happen here if we returned 'unknown' and then tried to process that when updating the snapshot result? We've been leaning towards returning an empty cell for when we can't find data in airtable so my instinct would be to return `null` here.